### ISSUE TITLE
Checksum of v0.6.0 release has changed

### ISSUE THREAD

**Issue by leonklingele** [koalaman/shellcheck#1407]
Was the release tarball of v0.6.0 modified on purpose?

```txt
Yesterday's SHA256sum:  4e6c46751f031095f33f46ad4a4cf51c79fd2398031260132d6b57390a59d293
SHA256sum as of today:  78f90aa8e618dc468bc1c36b6929216dc7a0c2679cd157e50919f7d8cc1899bc
```

See also https://github.com/Homebrew/homebrew-core/pull/34728.

**Comment by koalaman** [comment]
"On purpose" is an overstatement, but if it's one of the releases from the [storage bucket](https://shellcheck.storage.googleapis.com/index.html), then yes. Sorry about that. A quick post-mortem: 

1. It turns out I don't know git, and didn't realize a rebase wouldn't automatically update tags.
2. This caused the v0.6.0 tag to go out without 66b5f13c6ff in the stack
3. Travis-CI completed the first build and applied the v0.6.0 tag to it
4. I discovered the issue an hour later and set the tag correctly
5. Travis-CI failed on the second build due to a transient connection issue
6. I got the failure email later that evening and hit the Retry button
7. Travis-CI completed the second build and pushed it again with the same stable tag. This was about four hours after the first one.

So yes, all the v0.6.0 tagged tarballs in the storage bucket had different checksums for about four hours between 03:00 and 07:00 UTC 

The [Hackage tarball](http://hackage.haskell.org/package/ShellCheck-0.6.0) is not autopushed, was correct from the start, and should not have changed.

I'll be writing a script to sanity check release procedures, and this will definitely be included.
